real_world: true

figure_collector:
  group_num_instances: 7
  group_dist_thresh: 1.5
  use_local_coords: true

nn_model_path: nn_models/n_20ep.pt
color_detection_config_file: config/color_detection.json 

#additional settings
upload_data: true